src/openrct2/windows/tile_inspector.c

```diff
@@ -481,7 +485,9 @@ static void window_tile_inspector_mousedown(int widgetIndex, rct_window *w, rct_
 static void window_tile_inspector_update(rct_window *w);
 static void window_tile_inspector_dropdown(rct_window *w, int widgetIndex, int dropdownIndex);
 static void window_tile_inspector_tool_update(rct_window* w, int widgetIndex, int x, int y);
+static void window_tile_inspector_update_selected_tile(rct_window *w, int x, int y);
```

Comment:
The surrounding functions are all passed as to the window as event, but this one is not. Move this down to where `window_tile_inspector_set_page` begins.